pulsar-broker/src/test/java/org/apache/pulsar/client/api/AuthorizationProducerConsumerTest.java

```diff
@@ -427,6 +435,8 @@ public void testClearBacklogPermission() throws Exception {
         superAdmin.namespaces().grantPermissionOnNamespace(namespace, subscriptionRole,
                 Sets.newHashSet(AuthAction.consume));
         // now, subscriptionRole have consume authorization on namespace, so it will successfully clear backlog
+        assertEquals(sub1Admin.topics().getPartitionedTopicList(namespace),
```

Comment:
Could you please help add a test for the tenant admin can also get partitioned topic list?

Reply:
done, ptal